app/views/shared/_newsletter.html.haml

```diff
@@ -1,32 +1,34 @@
+- content_for :head do
+  :javascript
+    (function(w, d, t, h, s, n) {
+      w.FlodeskObject = n;
+      var fn = function() {
+        (w[n].q = w[n].q || []).push(arguments);
+      };
+      w[n] = w[n] || fn;
+      var f = d.getElementsByTagName(t)[0];
+      var v = '?v=' + Math.floor(new Date().getTime() / (120 * 1000)) * 60;
+      var sm = d.createElement(t);
+      sm.async = true;
+      sm.type = 'module';
+      sm.src = h + s + '.mjs' + v;
+      f.parentNode.insertBefore(sm, f);
+      var sn = d.createElement(t);
+      sn.async = true;
+      sn.noModule = true;
+      sn.src = h + s + '.js' + v;
+      f.parentNode.insertBefore(sn, f);
+    })(window, document, 'script', 'https://assets.flodesk.com', '/universal', 'fd');
+
 .row.justify-content-md-center
   .col-md-8
     %h2= t('homepage.newsletter.title')
     %p= t('homepage.newsletter.description')
 
-    -# Mailchimp Signup Form
-    #mc_embed_signup
-      %form.validate#mc-embedded-subscribe-form{ action: "https://codebar.us8.list-manage.com/subscribe/post?u=b4652d85b385945c79f2ffa2e&amp;id=3798974cb3",
-                                                 method: "post",
-                                                 name: "mc-embedded-subscribe-form",
-                                                 target: "_blank",
-                                                 novalidate: true }
-        #mc_embed_signup_scroll
-          #indicates-required
-          .mc-field-group.mb-3
-            .d-flex.justify-content-between
-              %label.form-label{ for: "mce-EMAIL" }
-                Email Address
-                %span.asterisk *
-              %small
-                %span.asterisk * indicates required
-            %input.required.email.form-control#mce-EMAIL{ type: 'email', value: '', name: 'EMAIL' }
-
-          .clear#mce-responses
-            .response#mce-error-response{ style: 'display:none' }
-            .response#mce-success-response{ style: 'display:none' }
-
-          -# real people should not fill this in and expect good things - do not remove this or risk form bot signups
-          %div{ style: 'position: absolute; left: -5000px;', 'aria-hidden': 'true' }
-            %input{ type: 'text', name: 'b_b4652d85b385945c79f2ffa2e_3798974cb3', tabindex: '-1', value: '' }
-          .d-flex.justify-content-between.align-items-center
-            %input.btn.btn-primary#mc-embedded-subscribe{ type: 'submit', value: 'Subscribe', name: 'subscribe' }
+    -# Flodesk signup form
+    #fd-form-678b693a7ae9331608185173
+      :javascript
+        if (window.fd) window.fd('form', {
+          formId: '678b693a7ae9331608185173',
+          containerEl: '#fd-form-678b693a7ae9331608185173'
+        });
```

Comment:
Before release, this snippet has to come from codebar’s Flodesk account (right now this came from my test one).

Reply:
Update: done